.github/actions/deploy/action.yaml

```diff
@@ -64,12 +60,10 @@ runs:
       run: npm run build -w nextjs-website
 
     - name: Configure AWS Credentials
-      env:
-        AWS_REGION: eu-south-1
-      uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838
+      uses: ./.github/actions/configure-aws-credentials
       with:
-        role-to-assume: ${{ inputs.iam_role }}
-        aws-region: ${{ env.AWS_REGION }}
+        aws_region: 'eu-south-1'
```

Comment:
Can the action take the `aws_region` from env if defined? Otherwise it uses `eu-south-1` as default.